docs/explanation/stable-release-updates.rst

```diff
@@ -1,2 +1,257 @@
+:ref:`release <ArchivePockets_Release>`,
+:ref:`security <ArchivePockets_Security>`, and
+:ref:`updates <ArchivePockets_Updates>`.
+It shouldn't include other packages from
+:ref:`proposed <ArchivePockets_Proposed>` or
+:ref:`backports <ArchivePockets_Backports>`,
+with one exception: other packages built from the affected source package must
+be updated if they're generally installed.
+
+If the fix is sufficient, the :term:`SRU Verification Team` will update the bug status
+to ``In Progress``, and change the  ``verification-needed-$RELEASE`` tag to
+``verification-failed-$RELEASE``. If the fix is sufficent, the :term:`SRU Verification Team`
+will tag it as ``verification-done-$RELEASE``.
+
+If you encounter a regression in a package uploaded to
+:ref:`proposed <ArchivePockets_Proposed>`, do the following:
+
+1. File a bug report describing the nature of the regression.
+#. Tag the bug as ``regression-proposed``.
+#. `Ask a bug supervisor`_ to target the bug to the appropriate :ref:`Ubuntu releases <UbuntuReleases>`.
+#. Follow up on the :term:`SRU` bug report referenced from the package changelog, pointing
+   to the new bug. If there is more than one bug in the :term:`SRU` changelog, follow up to
+   the bug that is most closely related to the regression.
+#. Set the ``verification-failed-$RELEASE`` tag on the corresponding :term:`SRU` bug report.
+
+.. note::
+   ``$RELEASE`` represents the release name of your upload.
+
+Packages accepted into :ref:`proposed <ArchivePockets_Proposed>` automatically trigger
+related :term:`autopkgtests <autopkgtest>`.
+
+If an :term:`SRU` upload triggers an :term:`autopkgtests <autopkgtest>` regression,
+the target package will not be released into :ref:`updates <ArchivePockets_Updates>` until
+the issue is resolved. Once the tests are completed, the pending :term:`SRU` page provides
+links to any failures noticed for the selected upload. It's the responsibility of the uploader
+or the person performing update verification to ensure that the upload doesn't cause any
+:ref:`regressions <Regressions>`, both in manual and automated testing.
+
+.. _Regressions:
+
+Regressions
+-----------
+
+Regressions are unintended negative consequences that updates introduce. They appear
+as new bugs or failures in previously well-functioning aspect of an
+:ref:`Ubuntu release <UbuntuReleases>`. 
+
+If a package update introduces a regression that makes it through the
+:ref:`verification <Verification>` process to :ref:`updates <ArchivePockets_Updates>`,
+file a bug report about the issue and add the tag ``regression-update`` to the bug.
+
+For regressions that only apply to the package in
+:ref:`proposed <ArchivePockets_Proposed>`, follow up on the bug with a detailed
+explanation and tag it with ``regression-proposed``.
+
+Regression tests
+~~~~~~~~~~~~~~~~
+
+To minimise the risk of :ref:`regressions <Regressions>` being introduced through
+:term:`SRU`, :term:`Canonical` will test each proposed kernel.
```

Comment:
nitpick: This page already established the acronym. There is no value in linking to the glossary.

```suggestion
SRU, :term:`Canonical` will test each proposed kernel.
```